legend-engine-xt-nonrelationalStore-mongodb-grammar/src/test/java/org/finos/legend/engine/language/mongodb/schema/grammar/from/TestValidExprQueryParser.java

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.language.mongodb.schema.grammar.from;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.finos.legend.engine.language.mongodb.schema.grammar.to.MongoDBQueryJsonComposer;
+import org.finos.legend.engine.protocol.mongodb.schema.metamodel.aggregation.DatabaseCommand;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Objects;
+
+@RunWith(Parameterized.class)
+public class TestValidExprQueryParser
+{
+
+    private static final org.slf4j.Logger LOGGER = org.slf4j.LoggerFactory.getLogger("Alloy Execution Server");
+    private final String inputJsonFile;
+
+    public TestValidExprQueryParser(String inputQueryFile)
+    {
+        inputJsonFile = inputQueryFile;
+    }
+
+    @Parameterized.Parameters(name = "{index}: {0}")
+    public static Collection<Object[]> data()
+    {
+        return Arrays.asList(new Object[][]{
+                {"json/exprquery/match_and_query.json"},
+                {"json/exprquery/match_eq_query.json"},
+
+        });
+    }
+
+    public ObjectMapper getObjectMapper()
```

Comment:
you end duplicating this on each test.  Should not you use the same configuration you will have in production?

Reply:
Agreed, moved to the parser's mapper